api/app/views/mno_enterprise/jpi/v1/app_instances/_resource.json.jbuilder

```diff
@@ -8,6 +8,15 @@ json.oauth_keys_valid app_instance.oauth_keys_valid
 #json.microsoft_trial_url app_instance.microsoft_trial_url
 json.created_at app_instance.created_at
 
+if app_instance.per_user_licence?
+	json.per_user_licence app_instance.per_user_licence
+	json.licences_count app_instance.active_licences_count
+end
+
+if app_instance.under_free_trial?
+  json.free_trial_end_at time_ago_in_words(app_instance.free_trial_end_at)
```

Comment:
So I understand we would do
```
json.free_trial_end_at app_instance.free_trial_end_at
```
and then translate in the front end. Am I right?

@alexnoox  Do we have any example in our code using "moment" + "i18n"?